msix/ubuntu-pro-agent-launcher/main.cpp

```diff
@@ -30,6 +30,9 @@ std::filesystem::path thisBinaryDir() {
 }
 
 int WINAPI wWinMain(HINSTANCE, HINSTANCE, PWSTR pCmdLine, int) try {
+  // Request to restart if closed for installing updates.
+  RegisterApplicationRestart(
+      NULL, RESTART_NO_CRASH | RESTART_NO_HANG | RESTART_NO_REBOOT);
```

Comment:
Should we pass the same command-line arguments as we were invoked with?

Reply:
That's what the NULL argument does: reuse the previous command line.

Reply:
Oh boy! I completely misread that statement! Thanks for catching my mistake!